README.md

```diff
@@ -172,40 +172,60 @@ quality_report(folder_sequences = directory_sequences,
 
-Default parameters for B cell receptors in macaques: 
+Default parameters for B cell receptors in macaques and humans as meantioned above: 
 
 ```r
-# `raw_length` = 400 
+# `raw_length` = 343 # based on your expected amplicon length
 # `trim_start` = 50 
 # `trim_finish` = 409
 # `trimmed_mean_quality` = 30
-# `posvalue_probe1` = 600 
-# `posvalue_probe2` = 400 
 # `cdr3_start` = 100
 # `cdr3_end` = 150
 
 ```
 
-Default parameters for T cell receptors in mice, mus musculus: 
+Default parameters for gamma delta T cell receptors in mice (_Mus musculus_): 
 
 ```r
-# `trim_start` = 200 
+# `raw_length` = 200 
 # `trim_finish` = 250
-# `trimmed_mean_quality` = 30
 
 ```
 
 
-Default parameters for Y cell receptors in zebrafish / other species / when uncertain:
+Default parameters for when uncertain:
+
+For other species not listed or if you are uncertain, a `trimmed_mean_quality` or minimum phred quality score = 30 should be used as this returns an average of 99.9% base call accuracy for the analyzed samples.
 
 ```r
 # `trimmed_mean_quality` = 30
 
 ```
 
+If you are uncertain of your amplicon length, first check the quality metrics of your sequences using `summarise_quality`. Save the results in an object to evaluate the quality metrics columns, especially `raw.length` and `trim.finish` columns, such as the mean, min, and max values. This will help you to identify which parameters you should change.
+
+```r
+quality_control_summary <- summarise_quality(folder_sequences = directory_sequences,
+                                         secondary.peak.ratio = 0.33,
+                                         trim.cutoff = 0.01,
+                                         )
+
+```
 
+If you are still uncertain you can use a broad default parameters that will filter mostly on mean base call accuracy. To do that, change the `raw_length` to 0 and and `trim_finish` to 50. This will null the filter based on length but the base mean accuracy  of 99.9% will be maintained. You can also put `cdr3_start` and `cdr3_end` to 0 if there is not position that you are particular interested in having with higher accuracy.
+
+```r
+quality_report(folder_sequences = directory_sequences,
+               outputfile = "QC_report.html",
+               output_dir = "~/Desktop/project_folder/results_output",
+               folder_path_fcs = directory_flowdata,
+               probe1 = "PE.Cy7.A", probe2 = "Alexa.Fluor.700.A",
```

Comment:
remove 224-225 or change it to something like "probe1 = "my_probe_channel_1"

Reply:
Addressed in commit 8714bee89b7765008e7b9cde4e08dd037a47482e